docs/AutomationProtocols.md

```diff
@@ -0,0 +1,98 @@
+---
+id: automationProtocols
+title: Automation Protocols
+---
+
+With WebdriverIO you can choose between multiple automation technologies when running your e2e tests locally or in the cloud. By now almost all modern browser support next to [WebDriver](https://w3c.github.io/webdriver/) also another native interface called [DevTools](https://chromedevtools.github.io/devtools-protocol/) that can be used for automation purposes. Both have advantages and disadvantages depending on your use case and environment.
```

Comment:
With WebdriverIO you can choose between multiple automation technologies when running your e2e tests locally or in the cloud. By now almost all modern browsers support the [WebDriver](https://w3c.github.io/webdriver/) interface next to another native interface called [DevTools](https://chromedevtools.github.io/devtools-protocol/) that can be used for automation purposes. Both have advantages and disadvantages depending on your use case and environment.